ext/tidy/tidy.stub.php

```diff
@@ -2,59 +2,59 @@
 
 /** @generate-function-entries */
 
-function tidy_parse_string(string $input, array|string|null $config_options = null, ?string $encoding = null): tidy|false {}
+function tidy_parse_string(string $string, array|string|null $config = null, ?string $encoding = null): tidy|false {}
```

Comment:
I was a bit unsure about `$config` vs `$options` here. I went with `$config`, because this parameter accepts either a path to a config file, or a config options array.